### PR TITLE
add sprkstep examples

### PR DESCRIPTION
**Pull request by balos1**
Change to point to main after staging is merged.

### CODE REVIEW

**Review by drreynolds — APPROVED** [review]

File: linux-ubuntu20.04-x86_64/gcc-9.4.0/double/ark_harmonic_symplectic.out
Comment (gardner48) [inline linux-ubuntu20.04-x86_64/gcc-9.4.0/double/ark_harmonic_symplectic.out:1]: Does the GitHub CI diff outputs? The formatting here is different than in the updated PR, specifically `err`.

Comment (balos1) [reply linux-ubuntu20.04-x86_64/gcc-9.4.0/double/ark_harmonic_symplectic.out:1]: I am unsure how the CI is passing with the double answers not being updated.

Comment (gardner48) [reply linux-ubuntu20.04-x86_64/gcc-9.4.0/double/ark_harmonic_symplectic.out:1]: I think the comparison script extracts the values and computes either the relative or absolute change. So the formatting might not actually matter if the change is "sufficiently small."

Comment (balos1) [reply linux-ubuntu20.04-x86_64/gcc-9.4.0/double/ark_harmonic_symplectic.out:1]: Makes sense. I'm updating them, but the easiest way is to break them first.

**Review by gardner48 — APPROVED** [review]

**Pull request merged**